Marlin/Configuration_adv.h

```diff
@@ -847,47 +847,59 @@
-  #define PAUSE_PARK_NOZZLE_TIMEOUT 45        // Turn off nozzle if user doesn't change filament within this time limit in seconds
-  #define FILAMENT_CHANGE_NUMBER_OF_ALERT_BEEPS 5 // Number of alert beeps before printer goes quiet
-  #define PAUSE_PARK_NO_STEPPER_TIMEOUT       // Enable to have stepper motors hold position during filament change
-                                              // even if it takes longer than DEFAULT_STEPPER_DEACTIVE_TIME.
-  //#define PARK_HEAD_ON_PAUSE                // Go to filament change position on pause, return to print position on resume
-  //#define HOME_BEFORE_FILAMENT_CHANGE       // Ensure homing has been completed prior to parking for filament change
+#define SMART_PAUSE
+#if ENABLED(SMART_PAUSE) 
+  //Move settings
+  #define PAUSE_X_POS        X_MAX_POS // Not defined = Current X position = Only Y move
+  //#define PAUSE_Y_POS      3 // Not defined = Current Y position = Only X move
+  #define PAUSE_Z_ADD        2 // Z addition of hotend (lift)
+  #define PAUSE_XY_F        150 // X and Y axes feedrate in mm/s (also used for delta printers Z axis)
+  #define PAUSE_Z_F          12 // Z axis feedrate in mm/s (not used for delta printers)
+  //Extrusion settings
+  #define PAUSE_RET_F        RETRACT_FEEDRATE // Initial retract feedrate in mm/s
+  #define PAUSE_RET           2 // Initial retract in mm
+  #define PAUSE_UNLOAD       70 // 0  for manual unloading
+  #define PAUSE_UNLOAD_F     50 // Unload filament feedrate in mm/s - filament unloading can be fast
+  #define PAUSE_LOAD         70 // Load filament length over hotend in mm
+  #define PAUSE_LOAD_F       15 // Load filament feedrate in mm/s - filament loading into the bowden tube can be fast
+  #define PAUSE_EXTRUDE_F   6.6 // Extrude filament feedrate in mm/s - must be slower than load feedrate
+  #define PAUSE_EXTRUDE      20 // Purge/load - 0 to disable for manual extrusion
+  //Blowing
+  #define PAUSE_FANSPEED    120//Blowing to cold the purged filament
+  #define PAUSE_FAN           0
+  #define PAUSE_DWELL    4*1000 //Time to cold the purged filament#if ENABLED(SINGLENOZZLE)
+  // Spool_Swap
+  #define PAUSE_SPOOL_SWAP 
+  #if ENABLED(PAUSE_SPOOL_SWAP)
+    #define SPOOL_SWAP_UNLOAD   RETRACT_LENGTH_SWAP // Don't eject filament because print continue
+    #define SPOOL_SWAP_LOAD     RETRACT_LENGTH_SWAP // Same as FWRETRACT if enabled
+    #define SPOOL_SWAP_EXTRUDE  20 //Purge length
+  #endif 
+  //Miscillaneous 
+  #define PAUSE_TIMEOUT     120 // Turn off nozzle if user doesn't change filament within this time limit in seconds
+  #define PAUSE_BEEPS         1 // Number of alert beeps before printer goes quiet
+  #define PAUSE_STEPPER_ON      // Enable to have stepper motors hold position during filament change
+  #define PARK_HEAD_ON_PAUSE    // Go to filament change position on pause, return to print position on resume
+  #define PAUSE_HOME_BEFORE     // Ensure homing has been completed prior to parking for filament change
 #endif
```

Comment:
TOTAL RENAMING
because ' advanced pause feature ' is not serious , too long 
Now all is short and very clear 
Some comments are deleted because not usefull , now it's compact and clear
Sorry for the renaming , but for the future of marlin , you must do t
Pause system , works fine now , and it's a strong feature